### ISSUE TITLE
About DFC2018_HSI

### ISSUE THREAD

**Issue by wang971010** [eecn/Hyperspectral-Classification#18]
I download the GRSS2018 dataset from https://hyperspectral.ee.uh.edu/?page_id=1075. But where is the HSI data 2018_IEEE_GRSS_DFC_HSI_TR.HDR in this dataset?

**Comment by eecn** [comment]
Yes, the data on the official website is no longer valid. If you need, you can contact me by email.

**Comment by yudadabing** [comment]
> I download the GRSS2018 dataset from https://hyperspectral.ee.uh.edu/?page_id=1075. But where is the HSI data 2018_IEEE_GRSS_DFC_HSI_TR.HDR in this dataset?

Can you share 2018_IEEE_GRSS_DFC_HSI_TR.HDR with me?

**Comment by eecn** [comment]
You can contact me by email.
